### ISSUE TITLE
Unable to update some Entity in python 3.4

### ISSUE THREAD

**Issue by victorien88** [ponyorm/pony#121]
I migrated one of my project to python 3.4 and since then I'm unable to modify some of my Entities. Eachtime I get :
Traceback (most recent call last):
  File "/home/ingest/Mam_Manager_rewrite/modules/models.py", line 1098, in setData
    cell.name = value
  File "<string>", line 2, in __set__
  File "/usr/local/lib/python3.4/site-packages/pony/utils.py", line 88, in cut_traceback
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/pony/orm/core.py", line 1449, in __set__
    cache.update_composite_index(obj, attrs, prev_vals, new_vals, undo)
  File "/usr/local/lib/python3.4/site-packages/pony/orm/core.py", line 1099, in update_composite_index
    if prev_vals is not None: del cache_index[prev_vals]
KeyError: ('JOUR1 - GROWTH PANEL',)

The entity is declared this way :+1: 
class Collection(db.Entity):
    id = PrimaryKey(int,sql_type='BIGINT',auto=True)
    name = Required(unicode,unique=True)
    comment = Optional(unicode)
    child = Set("ProjectGroup")

**Comment by kozlovsky** [comment]
The traceback says about composite index, but I don't see any composite index inside the entity definition. Did entity have any composite index?

**Comment by narcotix88** [comment]
Nop the strange is I have no composite index. I never used it.

**Comment by kozlovsky** [comment]
Thanks for the reporting, I was able to reproduce the bug. Will investigate into it.

**Comment by kozlovsky** [comment]
Fixed. The bug was introduced in release 0.6.1. The fix will be part of the new release 0.6.2. Right now you can take the latest PonyORM version from the GitHub or roll back to release 0.6

**Comment by socketpair** [comment]
also example of program that reproduce that bug:

```
from pony.orm import db_session, Database, Required
database = Database()
class Tbl(database.Entity):
    f1 = Required(str, unique=True)
database.bind('sqlite', ':memory:', create_db=True)
database.generate_mapping(create_tables=True)
with db_session:
    xxx = Tbl(f1='tmp_peer_config')
    xxx.f1 = 'asd'
```

**Comment by socketpair** [comment]
I cannot rollback to 6.0 since it have another annoying bug: when `.exists` return `True` on empty database. (Cannot find that issue on github)

**Comment by kozlovsky** [comment]
@socketpair did you mean that the bug is not fixed yet? The latest repository code behave well in my testing. Or you mean that you are awaiting official release with this bug fixed?

`exists` bug was not reported on the GitHub, it was fixed in commit 17db84e4
